### PR TITLE
Added doc for [Focus]

### PR DESCRIPTION
**Pull request by FrankBakkerNl**

### CODE REVIEW

File: docs/advanced/DevWorkflow.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+id: DevWorkflow
```
Comment (helto4real) [inline docs/advanced/DevWorkflow.md:2]: You need to add the page in the menu system here < https://github.com/net-daemon/docs/blob/main/sidebars.js#L34>

**Review by helto4real — CHANGES_REQUESTED** [review]

**Review by helto4real — APPROVED** [review]

File: sidebars.js [hunk 2]
```diff
@@ -32,7 +32,8 @@ module.exports = {
 
     ],
     Advanced: [
-      'advanced/thread_safety'
+      'advanced/thread_safety',
+      'advanced/dev_workflow.md'
```
Comment (helto4real) [inline sidebars.js:36]: Should be the id, DevWorkflow

**Review by helto4real — CHANGES_REQUESTED** [review]

File: sidebars.js (hunk 2 above)
Comment (FrankBakkerNl) [reply sidebars.js:36]: Ah, I renamed the file to snake_casing wil update the id as wel

**Review by helto4real — APPROVED** [review]
Now it works. A cool thing is in the CI there is a preview.

**Comment by helto4real** [comment]
I will merge this next release

File: docs/advanced/dev_workflow.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+title: Development workflow
+---
+
+# Development workflow
+
+## Test individual applications in development environment
+
+Over time a single NetDaemon solution might contain a large number of applications. While you are working on one NetDaemon app in your development environment you might not want to run all the NetDaemon applications each time you run it for testing.
+
+You can do this by applying  the `[Focus]` Attribute to the app class(es) you want to test. If one or more apps have this attribute set NetDaemon will only launch these applications and ugnore all others.
```
Comment (BeeHiveJava) [inline docs/advanced/dev_workflow.md:12]:
```suggestion
You can do this by applying  the `[Focus]` Attribute to the app class(es) you want to test. If one or more apps have this attribute set NetDaemon will only launch these applications and ignore all others. 
```

**Pull request merged**